pkgs/top-level/perl-packages.nix

```diff
@@ -15106,6 +15106,28 @@ let
     doCheck = false; /* wants network */
   };
 
+  NetHTTPSNB = buildPerlPackage {
+    pname = "Net-HTTPS-NB";
+    version = "0.15";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/O/OL/OLEG/Net-HTTPS-NB-0.15.tar.gz";
+      sha256 = "0kwc4z8pqnbc396wjnlgdmri10zdh91f2bi6saxkpfjzlm7wysba";
+    };
+    propagatedBuildInputs = [ URI ];
+    __darwinAllowLocalNetworking = true;
+
+    # requires network
+    doCheck = false;
+
```

Comment:
After including IOSocketSSL, tests pass on Linux. The tests are against a listening socket on localhost.

I don't have a Darwin machine at hand, but I don't think `__darwinAllowLocalNetworking` is required for this sort of test.

```suggestion
```